src/ensembl/production/metadata/updater/core.py

```diff
@@ -60,7 +63,7 @@ def get_meta_list_from_prefix_meta_key(self, species_id, prefix):
         result_dict = {k: v for k, v in species_meta.items() if k.startswith(prefix)}
         return result_dict
 
-    def process_core(self, **kwargs):
+    def process_core(self, force=False, **kwargs):
```

Comment:
force is typically a param to pass to the constructor and kept all along with `self.force`.